plugins/org.eclipse.fordiac.ide.product/org.eclipse.fordiac.ide.product.target

```diff
@@ -29,42 +29,42 @@
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="http://download.eclipse.org/modeling/emf/compare/updates/releases/3.3"/>
-			<unit id="org.eclipse.emf.compare.egit.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.egit.source.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.ide.ui.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.ide.ui.source.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.rcp.ui.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.rcp.ui.source.feature.group" version="3.3.24.202401051648"/>
-			<unit id="org.eclipse.emf.compare.source.feature.group" version="3.3.24.202401051648"/>
+			<repository location="http://download.eclipse.org/modeling/emf/compare/updates/releases/3.3/R202406060900"/>
+			<unit id="org.eclipse.emf.compare.egit.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.egit.source.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.ide.ui.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.ide.ui.source.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.rcp.ui.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.rcp.ui.source.feature.group" version="3.3.25.202406060900"/>
+			<unit id="org.eclipse.emf.compare.source.feature.group" version="3.3.25.202406060900"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<repository location="https://download.eclipse.org/nattable/releases/2.3.0/repository/"/>
 			<unit id="org.eclipse.nebula.widgets.nattable.core.feature.feature.group" version="2.3.0.202402191815"/>
 			<unit id="org.eclipse.nebula.widgets.nattable.core.source.feature.feature.group" version="2.3.0.202402191815"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/releases/2024-03"/>
-			<unit id="javax.activation" version="1.2.2.v20221203-1659"/>
-			<unit id="org.eclipse.egit.feature.group" version="6.9.0.202403050737-r"/>
-			<unit id="org.eclipse.emf.sdk.feature.group" version="2.37.0.v20240203-1010"/>
-			<unit id="org.eclipse.ocl.all.sdk.feature.group" version="5.20.0.v20240305-1416"/>
+			<repository location="https://download.eclipse.org/releases/2024-06"/>
+			<unit id="javax.activation" version="1.1.0.v201211130549"/>
```

Comment:
Why did you downgrade from `javax.activation` version `1.2.2` to `1.1.0`?

Reply:
because I was stupid when I searched for the version delivered with https://download.eclipse.org/releases/2024-06/202406121000/plugins/
Will put it back to 1.2.2.

Reply:
done